src/parser/rogue/assassination/modules/features/AlwaysBeCasting.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { When } from 'parser/core/ParseResults';
+// import SPELLS from 'common/SPELLS';
+import CoreAlwaysBeCasting from 'parser/shared/modules/AlwaysBeCasting';
+import { formatPercentage } from 'common/format';
+import { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+// import SpellLink from 'common/SpellLink';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+//   get downtimeSuggestionThresholds() {
+//     return {
+//       actual: this.downtimePercentage,
+//       isGreaterThan: {
+//         minor: 0.4,
+//         average: 0.45,
+//         major: 0.5,
+//       },
+//       style: ThresholdStyle.PERCENTAGE,
+//     };
+//   }
+// }
+
+// TODO: Change the following to be Ass Rogue specific.
+// If a fight requires movement, try saving <SpellLink id={SPELLS.PRESENCE_OF_MIND.id} /> to make your next 2 <SpellLink id={SPELLS.ARCANE_BLAST.id} /> casts instant.
+
```

Comment:
You should uncomment that and use downtimeSuggestionThresholds instead of magic numbers in the suggestions